### DISCUSSION TITLE
Certificate of completion

### DISCUSSION THREAD

**Discussion by emoloic** [mrdbourke/pytorch-deep-learning#154] [Q&A] [answered]
Hello !!!
I wanted to know whether is there a chance to get a certificate after watching the video on YouTube.
Thanks.

**Comment by RCHRD-JHN** [comment]
Hi! I'm also curious about this topic. Waiting for the answer. Thanks!

**Comment by mrdbourke** [comment, marked as answer]
Hey @emoloic,

There is no certificate upon completion of the video on YouTube, however, there is a certificate at the end of the course on either Zero to Mastery or Udemy.

This is because the YouTube video is the first ~25 hours of the course where as the full course is close to 50 hours.

See more here: https://dbourke.link/ZTMPyTorch

**Reply by emoloic** [reply]
Thanks.

**Comment by RCHRD-JHN** [comment]
Thanks for answering my question. I already signed up yesterday in Udemy
the moment I saw that lesson hehe. Thanks!
